pandas/tests/frame/test_reshape.py

```diff
@@ -49,14 +48,6 @@ def test_pivot(self):
         assert pivoted.index.name == 'index'
         assert pivoted.columns.names == (None, 'columns')
 
-        with catch_warnings(record=True):
-            # pivot multiple columns
-            simplefilter("ignore", FutureWarning)
-            wp = tm.makePanel()
-            lp = wp.to_frame()
-            df = lp.reset_index()
-            tm.assert_frame_equal(df.pivot('major', 'minor'), lp.unstack())
```

Comment:
would this test not need to replaced with a similar test not dependent on tm.makePanel?